index.html

```diff
@@ -12,6 +12,44 @@
 
   <script type="application/json" id="kuma-config"><%- config %></script>
   <script type="module" src="/src/main.ts"></script>
+  <style type="text/css">
```

Comment:
**Question**: Any reason these styles can’t live in a file imported in `main.scss`? Then we’d be able to use Sass variables.

Reply:
Because we want the loader to appear as soon as possible, actually we probably need to move these above everything else. Let me look at that.

Reply:
I've moved things around slightly here so the CSS is ordered before the JS script tag. I even tried inlining the gif, but it was a little cumbersome having it inline, maybe there's a vite plugin we can use to do that? If so that can be a improvement after this PR.

i think this one is ok to resolve, lmk if not and we can reopen.